test/module/irohad/ametsuchi/flat_file_test.cpp

```diff
@@ -16,6 +16,8 @@
  */
 
 #include "ametsuchi/impl/flat_file/flat_file.hpp"
+#include "ametsuchi/impl/flat_file/flat_file.cpp"
```

Comment:
Something went wrong here

Reply:
I need check_consistency function, it is declared in cpp file

Reply:
Thus maybe move this function declaration to the hpp?

Reply:
fixed